src/ui/media/MediaInfo.as

```diff
@@ -291,6 +293,9 @@ public class MediaInfo extends Sprite {
 		if (mycostume) {
 			result.width = mycostume.width();
 			result.height = mycostume.height();
+			if (mycostume.bitmapResolution != 1) {
+				result.bitmapResolution = mycostume.bitmapResolution;
+			}
```

Comment:
The `if` is here to shrink the JSON a bit when we don't need to store the resolution.